lib/models/os.coffee

```diff
@@ -304,6 +304,42 @@ getOsModel = (deps, opts) ->
 				throw new Error('No such version for the device type')
 			.asCallback(callback)
 
+	###*
+	# @summary Get an applications config.json
+	# @name getConfig
+	# @public
+	# @function
+	# @memberof resin.models.os
+	#
+	# @param {Number} Id - application id (number)
```

Comment:
This should probably be `nameOrId`, and disambiguate the two, like most of the application methods. Partly for consistency, and also because that's what you've assumed it is in the examples here :smile: